### ISSUE TITLE
New theme gets lost?

### ISSUE THREAD

**Issue by strengejacke** [posit-dev/positron#4791]
#### Positron and OS details:

Positron Version: 2024.09.0 (system setup) build 77
Code - OSS Version: 1.93.0
Commit: 9b6f7c066546a4c7f104368fc769fb3768b08bfd
Date: 2024-09-23T02:44:07.418Z
Electron: 30.4.0
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631
#### Interpreter details:

R 4.4.1

## Describe the issue:

There was a new dark theme added to Positron (https://github.com/posit-dev/positron/pull/4762 I guess), which seems to be used by default. When I change the theme via the settings, and want to change back to the new theme, it no longer appears in the list of installed/pre-defined themes:

![image](https://github.com/user-attachments/assets/d630b475-d927-4227-9b12-33b2c1ccea4b)

![image](https://github.com/user-attachments/assets/4f2750f1-ab23-48c4-bf87-56b2a56c1713)

(It's not crucial, I still prefer the "old" dark theme anyway ;-)

**Comment by jmcphers** [comment]
>  it no longer appears in the list of installed/pre-defined themes:

What were you expecting to see in this list that you didn't?

(it's true that the name of this theme changed from "Default Dark Modern" to "Default Positron Dark" in the latest build!)

**Comment by strengejacke** [comment]
When I initially started Positron after updating, the left panel (which is blue-grey in the screenshot above), it was black. All panes were completely black, so it looked like a completely different theme. When I then opened the settings for the color themes the first time, there was an entry in that list with a _different_ name (I _think_ it was _Dark Modern_). Then I switched to the former theme and applied it again, but after that, the (new?) _Dark Modern_ theme no longer appeared in that theme list, so I can't restore the "new" theme that I saw directly after installing and starting the latest Positron update. That's why I can't reproduce this issue.
